hack/lib/common.sh

```diff
@@ -457,7 +457,7 @@ function kube::common::start_workload_controller_manager {
 function kube::common::start_controller_manager {
     local controller_opts="*,-nodelifecycle,-nodeipam"
     if [ "${IS_RESOURCE_PARTITION}" == "true" ]; then
-       controller_opts="nodelifecycle,nodeipam"
+       controller_opts="nodelifecycle"
```

Comment:
looks like your code is not based on the most recent code

Reply:
yeah, this is merged and removed.